src/deriver/queue_payload.py

```
@@ -0,0 +1,92 @@
+                sender_name=sender_name,
+                target_name=target_name,
+                session_name=session_name,
+                message_id=message_id,
+                created_at=created_at,
+                task_type=task_type,
+            )
+            # Convert back to dict for compatibility with JSON serialization
+            # mode='json' ensures datetime is converted to ISO string
+            payload = validated_payload.model_dump(mode='json')
+        except Exception as e:
+            raise ValueError(f"Failed to create valid payload: {str(e)}") from e
```

Ruff (0.12.2)

Avoid specifying long messages outside the exception class

(TRY003)